programming-extensions/programming-extension-vfsprovider/src/main/java/org/objectweb/proactive/extensions/dataspaces/vfs/VFSFactory.java

```diff
@@ -100,7 +104,24 @@
      *             when initialization or configuration process fails
      */
     public static DefaultFileSystemManager createDefaultFileSystemManager() throws FileSystemException {
-        return createAndConfigureFileSystemManager(ManagerType.BOTH);
+        return createAndConfigureFileSystemManager(ManagerType.BOTH, null);
```

Comment:
Is it possible to have a **BOTH** DataSpace (i.e., PROACTIVE and NON_PROACTIVE)?  
Would you explain how please.

Reply:
The file system manager is able to resolve files according to several registered providers. It associate files with providers based on the url (e.g. sftp://, file://, pamr://). It is the main entry point of apache vfs. So yes, a manager is able to handle files coming from different (proactive or non-proactiive) providers

Reply:
Please note that VFSFactory creates an apache vfs  FileSystemManager, not a ProActive dataspace. Dataspaces are another layer.